DTS-STN/Alpha-Site
# Sr only close button for menu

# Description

Updated menu to have a close sr-only close button under 768px and added a state change to indicate menu closing. Also changed the onmenuClick function to work with a state now.
#228 

## Acceptance Criteria

Menu should have a close button when screen is under 768px width. The "menuDropwdown" class should have an aria-expanded indicating that it is closed or open.

## Test Instructions

1. Run the code locally and inspect put the screen width under 768px
2. Open the menu and tab to the close button
3. Check to see if the close button is invisible when not tabbed, and if the aria-expanded state changes in the console onClick.
4. Put the screen over 768px and try tabbing to the close button (It shouldn't work, if it does, it's now well implemented)

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
